hbase-server/src/main/java/org/apache/hadoop/hbase/tool/Canary.java

```diff
@@ -1629,7 +1684,7 @@ public static void main(String[] args) throws Exception {
     final Configuration conf = HBaseConfiguration.create();
 
     // Loading the generic options to conf
-    new GenericOptionsParser(conf, args);
+    //new GenericOptionsParser(conf, args);
```

Comment:
why aren't we parsing generic options anymore?

Reply:
This was redundant. We already do parsing in https://github.com/apache/hadoop/blob/branch-2.8/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/util/ToolRunner.java#L70